### ISSUE TITLE
R85 (85.0.4183.84) works fine for me

### ISSUE THREAD

**Issue by daemongloom** [sebanc/brunch#504]
HP Spectre X2.
Rammus image, updated 83>84>85.
Linux and Android apps also work fine.

**Comment by aia832003** [comment]
> HP Spectre X2.
> Rammus image, updated 83>84>85.
> Linux and Android apps also work fine.

Can you share the method and how to update? I'm not sure i quite follow how to make this work.

**Comment by CornishRattler** [comment]
Settings - about chrome - additional details - change channel - beta

**Comment by stucki** [comment]
I can confirm that R85 seems to work fine using Surface Go.

**Comment by CongoKong** [comment]
Agreed. Was able to upgrade hatch from 83->84->85 without any issues.

**Comment by steinano** [comment]
I upgraded from 83>84>85. The 85 upgrade broke the sound (Matebook X Pro).

**Comment by CongoKong** [comment]
For me the wallpaper app does not work anymore (it works with images from disk only). But I am not sure if this is related to the 84 or 85 upgrade.

**Comment by thadrumr** [comment]
For me SmartLock no longer works with version 85.  This worked in version 83 and 84.  This is with an HP Elitebook folio 9480m running Rammus image.

**Comment by retsimx** [comment]
Dell XPS 9570 is working fine with 85.

**Comment by chickenphaal** [comment]
85 broke camera (again) on HP Spectre X360 13" 2018. Reef has been the only image I can get it to work on as it had older camera drivers apparently. Mic still doesnt work though.
